src/extension.ts

```diff
@@ -92,47 +92,27 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 // this method is called when your extension is deactivated
-export function deactivate() {
-}
+export function deactivate() {}
 
+const _onSetLanguageClient = new vscode.EventEmitter<vslc.LanguageClient>()
+export const onSetLanguageClient = _onSetLanguageClient.event

```

Comment:
So far we have followed the convention to name any global variables `g_`. Probably best to follow that for now, just so that we don't have a mix of conventions. I think this would apply to the variables in the two lines here.

Reply:
okay, I just realize what `g_` stands for !
well, I'm okay to change `_onSetLanguageClient` to `g_onSetLanguageClient`, but imho changing `onSetLanguageClient` to `g_` prefixed sounds weird.
Most of event subscription APIs are defined under the jQuery's `on` naming convention, so I guess we're better to follow that and it deserves the exception of our convention ?

Reply:
Yes, agreed, that makes sense!